assets/schemas/money/latest

```diff
@@ -1 +1 @@
-v202201.00
\ No newline at end of file
+v202201.00
```

Comment:
.

Reply:
I haven't changed anything here. This file already existed before.